### DISCUSSION TITLE
responsive webpage for a better layout on a phone or tablet.

### DISCUSSION THREAD

**Discussion by TheRealRWJ** [baimard/gestion#326] [Ideas]
# Thanks
Thank for your effort for this great app.
Know it is very appreciated

# Feature request:

Can you please make the webpage responsive for a better layout on a phone or tablet.
Thanks in advance

**Comment by baimard** [comment]
Dear,

The page is responsive, but a minimum for width is required. it's due to 
informations shown in tables.

I know that I can improve ergonomi :)

Cordialement,
Benjamin AIMARD

Le 24/02/2023 à 18:44, Richard a écrit :

**Comment by TheRealRWJ** [comment]
Thanks for your swift response.
May an idea, if it be possible, to use a kind of card like record. 
Perhaps with a minimal and a detailed view like NC Deck uses

I will be looking forward to it

**Comment by baimard** [comment]
Convert in discussion, I like your idea. Maybe some developpers can join this project.

**Comment by TheRealRWJ** [comment]
OK. I'm sure you don´t want my coding level  in your project.
But How do I help you with a translation to dutch?
